test/Chainweb/Test/Orphans/Internal.hs

```diff
@@ -194,7 +194,9 @@ arbitraryBlockHashRecordVersionHeightChain v h cid
         <$> infiniteListOf arbitrary
     | otherwise = discard
   where
-    graph = chainGraphAt v (max 0 (h - 1))
+    graph
+        | h == genesisHeight v cid = chainGraphAt v h
```

Comment:
This is the actual fix: genesis blocks are treated different than other blocks during graph transitions. The adjacent dependencies of a genesis header are from the graph at the height of the header itself. For other headers the graph of the parent (i.e. the target of the dependency) is used